### PR TITLE
Verify burger menu

### PR DESCRIPTION
**Pull request by amyrchyk**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java [hunk 1]
```diff
@@ -97,4 +103,17 @@ public LoginForm openLoginForm() {
     public boolean isLogOutBtnPresent() {
 	return logOutBtn.isPresent();
     }
+
+    public boolean isPresentBurgerMenuBtn() {
+	return menuBtn.isPresent();
+    }
+
+    public HeaderMenuList openBurgerMenu() {
+	menuBtn.click();
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java:112]: add logger

File: src/main/java/com/qaprosoft/carina/demo/gui/components/HeaderMenuList.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    }
+
+    public CoveragePage openCoveragePage() {
+	coverageBtn.click();
+	return new CoveragePage(getDriver());
+    }
+
+    public ContactPage openContactPage() {
+	contactBtn.click();
+	return new ContactPage(getDriver());
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/gui/components/HeaderMenuList.java:102]: add logger

Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/HeaderMenuList.java]: done

File: src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java (hunk 1 above)
Comment (amyrchyk) [reply src/main/java/com/qaprosoft/carina/demo/gui/components/Header.java:115]: done

**Comment by vrudyk** [comment]
ok

**Pull request merged**